app/src/main/java/org/dharmaseed/android/NavigationActivity.java

```diff
@@ -238,11 +240,45 @@ public void onRefresh() {
         LocalBroadcastManager.getInstance(this).registerReceiver(new BroadcastReceiver() {
             @Override
             public void onReceive(Context context, Intent intent) {
-                Log.i("navigationActivity", "Received update broadcast");
+                Log.i(LOG_TAG, "Received update broadcast");
                 updateDisplayedData();
             }
         }, new IntentFilter("updateDisplayedData"));
 
+        listView.setOnScrollListener(new ListView.OnScrollListener() {
+            @Override
+            public void onScrollStateChanged(AbsListView view, int scrollState) {}
+
+            @Override
+            public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCount, int totalItemCount) {
+                Log.d(LOG_TAG, "scroll position changed: " + firstVisibleItem);
```

Comment:
In my testing, this logging call is fairly noisy when scrolling in the app; it can produce many similar messages, which might make finding other things in the logs more difficult. There is a similarly noisy logging call on line 33 of `TextViewFader.java`. I was wondering how you'd feel about removing these before merging? I'm sure they were very useful for development, but I'm wondering if they'll continue to have value in the future.